k9mail-library/src/test/java/com/fsck/k9/mail/K9LibRobolectricTestRunner.java

```diff
@@ -0,0 +1,20 @@
+package com.fsck.k9.mail;
+
+import org.junit.runners.model.InitializationError;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+public class K9LibRobolectricTestRunner extends RobolectricTestRunner {
+
+    public K9LibRobolectricTestRunner(Class<?> testClass) throws InitializationError {
+        super(testClass);
+    }
+
+    @Override
+    protected Config buildGlobalConfig() {
+        return new Config.Builder()
+                .setSdk(21)
```

Comment:
I don't know much about Robolectric testing really. Do you think this should:

a) Generally match the targetSdkVersion?
b) Generally match the minimumVersion?
c) Just be consistent?

Currently it's 21 and we're building for 22.

I assume there's scenarios where you want to test a particular function and so you need a certain version